lib/sources/node.js

```diff
@@ -33,12 +36,30 @@ NodeSource.prototype.update = function(done) {
     if (!res.text) return done(new Error('No response'), false);
     if (res.status !== 200) return done(new Error('Bad response'), false);
 
-    this._parse(res.text)
-    done(undefined, true);
+      responseText += res.text;
```

Comment:
The indentation seems wrong here; compare to previous code (each line should be indented additional 2 spaces, here it's 4).